### DISCUSSION TITLE
Data Representation Years

### DISCUSSION THREAD

**Discussion by dt-woods** [USEPA/standardizedinventories#141] [Q&A] [answered]
Just checking to see if there are plans for updating the data representation years in the 2020--2022 time frame. Work is underway on developing new electricity baselines for these years and I want to make sure I'm building the most relevant inventories possible. Thanks!

**Comment by bl-young** [comment, marked as answer]
With v1.1 we released several datasets through 2021 (DMR, eGRID, GHGRP, TRI). I can inquire when the NEI 2021 dataset (which would be interim) might be released, and when the 2022 updates might be expected for other datasets.

**Reply by bl-young** [reply]
My guess is we would not see NEI 2021 before the end of the calendar year.

**Reply by dt-woods** [reply]
I appreciate the update, Ben. Can I assume that the data representation table on the README is up-to-date?

**Reply by bl-young** [reply]
Yes it is
